Dockerfile

```diff
@@ -1,8 +1,7 @@
-FROM python:3.7.3-alpine
+FROM python:3.7.3-alpine AS base
```

Comment:
I would encourage you to use the non-Alpine Python docker image for this, since it has a bunch of downsides and the recent buster based default Python image is not a lot bigger than Alpine. More details: https://pythonspeed.com/articles/base-image-python-docker-images/

Reply:
Awesome, let me know if you need any help with the review!